scenario/RewardsScenario.ts

```diff
@@ -156,8 +156,8 @@ scenario(
     const baseScale = (await comet.baseScale()).toBigInt();
     const toBorrow = 1_000n * baseScale;
 
-    const [rewardTokenAddress, rescaleFactor] = await rewards.rewardConfig(comet.address);
-    const rewardToken = context.getAssetByAddress(rewardTokenAddress);
+    const [_, rescaleFactor] = await rewards.rewardConfig(comet.address);
+    const rewardToken = await context.getRewardToken();
```

Comment:
we could have `context.getRewardToken` return both the `rewardToken` instance and the `rescaleFactor`, which would eliminate the need to call `rewards.rewardConfig` on line 159

Reply:
`getRewardConfig`?

Reply:
i like it. added a `getRewardConfig` function context.